services/user.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
```

Comment:
The import `from django.contrib.auth.models import User` is redundant if you are using `get_user_model()` to retrieve the user model. Consider using `get_user_model()` throughout the code to ensure compatibility with custom user models.